### ISSUE TITLE
Allow to show new notifications for threads you are participating/mentioned in only

### ISSUE THREAD

**Issue by velias** [Daskiworks/ghwatch#9]
Additional filter to other filters (by repo and type) - shown in Drawer. GitHub API `participating` request param.

**Comment by rvraghav93** [comment]
This would be really nice to have especially as a filter that can be quickly switched on and off!

**Comment by JonnyTech** [comment]
Agreed. I like to show all notifications with those that I am participating in being highlighted or filtered. This feature will make the app very similar to GitHub's web interface and is the only thing preventing me from using this app as a replacement. Please consider implementing it.
